packages/fscomponents/src/lib/wallets/ApplePay.web.ts

```diff
@@ -1,29 +1,42 @@
 import { default as ApplePayBase } from './ApplePay.base';
 
+export interface ApplePaySession {
+  openPaymentSetup?: (identifier: string) => void;
+  canMakePaymentsWithActiveCard?: (identifier: string) => void;
```

Comment:
Per Apple's docs, both of these methods return Promise<boolean>.

https://developer.apple.com/documentation/apple_pay_on_the_web/applepaysession

Reply:
@bweissbart Type for returned value from methods 'openPaymentSetup' and 'canMakePaymentsWithActiveCard' was changed. Please check.